package.json

```diff
@@ -9,15 +9,23 @@
     "async": "~0.9.0",
     "aws-sdk": "^2.0.25",
     "babel": "^5.5.4",
+    "bl": "~1.0.0",
```

Comment:
Many of those are likely dependencies of our dependencies. IE, a module we require depends on another module. And since the module we require failed to install, it's dependencies also failed.

I will clean this up when I review your PR.